sphinxcontrib/confluencebuilder/builder.py

```diff
@@ -22,6 +22,7 @@
 from sphinx.errors import ExtensionError
 from sphinx.util import status_iterator
 from sphinx.util.osutil import ensuredir, SEP
+from sphinx.builders.html import StandaloneHTMLBuilder
```

Comment:
(optional) Re-ordering these imports into lexicographical order would be a nicety.

Reply:
Addressed comment. I was not sure if you wanted me to re-order all the imports, I did it, I hope this is ok. I used isort module.

Reply:
Interesting. I was not aware that PEP8 had a recommendation for import ordering. Personally, I am not a fan of this recommendation. I believe (aside from `__future__` requirements) that sorting all imports lexicographically is a much cleaner way to organize imports (for readability); and I never understood why PEP promoted all these extra blank lines.

That being said, maybe I should concede my own personal preferences here. I suspect the majority of Python developers prefer it this way. Also, since Sphinx itself follows this style, I assume it would be good for this extension to follow suit.